EIPS/eip-draft-extbalance.md

```diff
@@ -0,0 +1,37 @@
+---
+eip: 2604
+title: EXTBALANCE rename and conditional pricing
+author: wjmelements.eth <wjmelements@gmail.com> (@wjmelements)
+discussions-to: https://ethereum-magicians.org/t/eip-2604-conditional-extbalance-repricing/
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-04-16
+requires: 1884
+---
+
+## Simple Summary
+Renames `BALANCE` to `EXTBALANCE` and reduces `EXTBALANCE` cost when querying the current contract.
+
+
+## Abstract
+This EIP renames `BALANCE` to `EXTBALANCE`.
+The gas cost of `EXTBALANCE` when the parameter matches the current address is reduced to a value slightly higher than the cost of `SELFBALANCE`.
+
+
+## Motivation
+As the primary purpose of `EXTBALANCE` following EIP1884 is to query external accounts, the name is updated accordingly.
+
+`EXTBALANCE` existed prior to `SELFBALANCE` so it is used by several immutable contracts, including the Uniswap-V1 template, that predate the Istanbul hard fork.
+These contracts use `EXTBALANCE` to query their own balance.
+
+## Specification
+
+At block `N`, the `EXTBALANCE` operation changes from 700 gas to a conditional cost:
```

Comment:
The way this type of thing is worded in most EIPs is along the lines of:
> As of FORK_BLOCK_NUMBER, the `EXTBALANCE` (`0x##`) opcode will cost:
> - 10 if ...
> - 700 otherwise